test/integration/brook/storage/redis_test.exs

```diff
@@ -23,41 +25,67 @@ defmodule Brook.Storage.RedisTest do
     end
 
     test "will append the event to a redis list", %{redix: redix} do
-      event1 = Brook.Event.new(author: "bob", type: "create", data: %{one: 1})
-      event2 = Brook.Event.new(author: "bob", type: "update", data: %{one: 1, two: 2})
+      event1 = Brook.Event.new(author: "bob", type: "create", data: %{"one" => 1})
+      event2 = Brook.Event.new(author: "bob", type: "update", data: %{"one" => 1, "two" => 2})
```

Comment:
is it valuable to keep these going in as atom keys and coming out as string keys to catch if we ever accidentally get overly generous with a `atom_keys: true` flag somewhere we don't want to?